noir-projects/aztec-nr/aztec/src/keys/point_to_symmetric_key.nr

```diff
@@ -1,10 +1,10 @@
 use dep::protocol_types::{constants::GENERATOR_INDEX__SYMMETRIC_KEY, grumpkin_point::GrumpkinPoint, utils::arr_copy_slice};
-use dep::std::{hash::sha256, grumpkin_scalar::GrumpkinScalar, scalar_mul::variable_base_embedded_curve};
+use dep::std::{hash::sha256, grumpkin_scalar::GrumpkinScalar, embedded_curve_ops::multi_scalar_mul};
```

Comment:
I decided to go ahead and do the changes in out of noir-std as well (instead of exposing the same interface out of noir std) because the changes were very isolated since we don't use the functions in many places.